spec/models/ems_cluster_spec.rb

```diff
@@ -106,6 +109,9 @@
     it('#all_miq_templates')    { expect(@cluster.all_miq_templates).to    match_array [@template1, @template2] }
     it('#all_miq_template_ids') { expect(@cluster.all_miq_template_ids).to match_array [@template1.id, @template2.id] }
     it('#total_miq_templates')  { expect(@cluster.total_miq_templates).to eq(2) }
+    it('#hosts')  { expect(@cluster.all_hosts).to match_array [@host1, @host2] }
```

Comment:
Why are the `#hosts` examples calling `#all_hosts`?    How are the `#hosts` and `#all_hosts` examples different here and above?